src/core/client/utility/screenEffect.ts

```diff
@@ -0,0 +1,40 @@
+import * as natives from "natives";
+import { SCREEN_EFFECTS } from "../../shared/enums/ScreenEffects";
+
+export abstract class ScreenEffect {
```

Comment:
These are all static; so let's try to move away from this and go towards consts.

See: #299

Reply:
[f16bd15](https://github.com/Stuyk/altv-athena/pull/334/commits/f16bd15982cc5d7d7194af1030db89a71a212ef5)